test/decl/protocol/special/coding/class_codable_failure_diagnostics.swift

```diff
@@ -0,0 +1,140 @@
+// We expect target-swift-frontend to return a status code of 1 here (since it
+// will be outputting diagnostics). That will fail the test, so we have to ||
+// true to set the status code to 0. This necessitates passing the input to
+// FileCheck through an intermediate file.
```

Comment:
Oh, great! Searched through a bunch of existing tests to look for how this is normally done but I guess I'd missed that. Let me update.